mobile-app/app/screens/AppNavigator/screens/Balances/components/BalanceCard.tsx

```diff
@@ -70,24 +74,45 @@ export function BalanceCard ({
       handleOnPress: () => onButtonGroupChange(ButtonGroupTabKey.dTokens)
     }
   ]
+  const [tabButtonLabel, setTabButtonLabel] = useState('')
+  const { hasFetchedToken } = useSelector((state: RootState) => (state.wallet))
+  const [isCollapsed, setIsCollapsed] = useState<boolean>(false)
   const onButtonGroupChange = (buttonGroupTabKey: ButtonGroupTabKey): void => {
     if (buttonGroupOptions !== undefined) {
       buttonGroupOptions.setActiveButtonGroup(buttonGroupTabKey)
       buttonGroupOptions.onButtonGroupPress(buttonGroupTabKey)
+      setButtonLabel(buttonGroupTabKey)
     }
   }
 
-  const { hasFetchedToken } = useSelector((state: RootState) => (state.wallet))
-  const [isCollapsed, setIsCollapsed] = useState<boolean>(false)
+  function setButtonLabel (buttonGroupTabKey: ButtonGroupTabKey): void {
```

Comment:
nothing wrong here in terms of functionality, but let's use `const` for code consistency